package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
   "version": "2.0.0",
+  "engines": {
+    "yarn": "1.3.2",
```

Comment:
Better upgrade this yar engine requirement to the latest stable 1.12.1

Reply:
Will do